src/main/java/com/switchfully/parkshark_2024_10/division/dto/CreateDivisionDto.java

```diff
@@ -0,0 +1,28 @@
+package com.switchfully.parkshark_2024_10.division.dto;
+
+import com.switchfully.parkshark_2024_10.user.Director;
+import jakarta.validation.constraints.NotNull;
+
+public class CreateDivisionDto {
+    String name;
+    Director director;
```

Comment:
Are we passing through a Director class, or just a String or Integer referencing the Director?

In other words, are we sending a request like:
```json
{
  "name": "test",
  "director": 123,
  "originalCompanyName": "test company"
}
```
or
```json
{
  "name": "test",
  "director": "Lukas Flamel",
  "originalCompanyName": "test company"
}
```
or, as it is now:
```json
{
  "name": "test",
  "director": {
    "id": 123
  },
  "originalCompanyName": "test company"
}
```

I'm not sure the last one is the cleanest one. I would prefer a `director_id` property that takes a Long and maps it to a (hopefully) existing Director.
```json
{
  "name": "test",
  "director_id": 123,
  "originalCompanyName": "test company"
}
```